### PR TITLE
Add the creation of a ceilometer user in the host for allow connectin…

### PR DESCRIPTION
**Pull request by jlarriba**
…g to libvirt

### CODE REVIEW

File: roles/edpm_telemetry/tasks/deploy_ceilometer.yml [hunk 1]
```diff
@@ -30,6 +30,18 @@
...
+  ansible.builtin.import_role:
+    name: edpm_users
+  vars:
+    edpm_users_users:
+      # 42405 is matching with the uid and gid created by kolla in the ceilometer containers
+      - {"name": "ceilometer", "uid": "42405", "gid": "42405", "shell": "/sbin/nologin", "comment": "ceilometer user", "groups": "libvirt"}
+    edpm_users_extra_dirs: []
+  tags:
+    - edpm_users
+    - edpm_telemetry
```
Comment (SeanMooney) [inline roles/edpm_telemetry/tasks/deploy_ceilometer.yml:43]: i dont see any molecule test for this
feel like adding a quick assert to ensure that this is created properly?

**Review by SeanMooney — COMMENTED** [review]
over all this looks ok to me but it would be nice to add test coverage

**Comment by jlarriba** [comment]
> change looks fine. it would be nice to refactor this role to match the task organization of the roles in this repo as it relates to separating tasks by their functionality (configure, install, run, etc).

Taking this suggestion into Jira

**Comment by jlarriba** [comment]
recheck

**Review by slagle — APPROVED** [review]

**Comment by jlarriba** [comment]
recheck

**Pull request merged**